gpbft/chain.go

```diff
@@ -103,7 +107,7 @@ func (ts *TipSet) String() string {
 		return "<nil>"
 	}
 
-	return fmt.Sprintf("%d@%s", ts.Epoch, hex.EncodeToString(ts.Key))
+	return fmt.Sprintf("@%d", ts.Epoch)
```

Comment:
why remove the key?

Reply:
100x 64characters in logs are not useful (I might replace it with truncation).

Reply:
Hm. The issue is that it makes it harder to tell why/how nodes disagree (they'll often agree on the epochs). But yeah, truncating it should be good enough.